src/github_services.py

```diff
@@ -221,13 +239,14 @@ def create_discussion_comment(
 
     variables = {
         'org_name': org_name,
-        'repository': repo
+        'repository': repo_name
     }
 
     response = requests.post(
         GITHUB_GRAPHQL_URL,
         json={'query': query, 'variables': variables},
-        headers=_get_request_headers()
+        headers=_get_request_headers(),
+        timeout=15
```

Comment:
I suggest you make 15 into a named top-level constant and use that constant everywhere, so that it can be easily configured.

Reply:
done